xapian-core/include/xapian/termgenerator.h

```diff
@@ -141,6 +144,16 @@ class XAPIAN_VISIBILITY_DEFAULT TermGenerator {
      */
     void set_stemming_strategy(stem_strategy strategy);
 
+    /** Set the stopper strategy.
+     *
+     *  @param strategy The strategy to use - possible values are:
+     *   - STOP_NONE:     Don't ignore the word if it is a stop word.  
+     *   - STOP_IGNORE:   Ignore the word if it is a stop word. 
+     *   - STOP_INDEX_UNSTEMMED_ONLY: Ignore the stemmed form of the
+     *               word if it is a stop word.   
```

Comment:
I have edited the documentation of the function and changed STOP_IGNORE to STOP_ALL and STOP_INDEX_UNSTEMMED_ONLY to STOP_STEMMED_FORM.